app/core/services/auth/auth.service.js

```diff
@@ -15,7 +15,7 @@ angular
       }
 
       function signIn({ email, password }) {
-        const queryUrl = `http://localhost:3000/heroes?email=${email}&password=${password}`  
+        const queryUrl = API_ENDPOINTS.HEROES + `?email=${email}&password=${password}`  
```

Comment:
I think it's great you're standardizing the API's endpoints by creating global variables to it. However, I think this PR does not aims on code quality in general and as you can see you refactored a peace of code that has nothing to do with the implementation tasks. Again, I think that changing was great, my only concern is to avoid possible git conflicts given that the team is not expecting that PR to change `auth service.`